src/rules/gameCards.js

```diff
@@ -0,0 +1,243 @@
+import lola from './cardPicture/lola.png';
+import leo from './cardPicture/leo.png';
+import carol from './cardPicture/carol.png'
+import enzo from './cardPicture/enzo.png';
+import jimbo from './cardPicture/jimbo.png';
+import stephania from './cardPicture/stephania.png';
+import daisy from './cardPicture/daisy.png';
+import jayce from './cardPicture/jayce.png';
+import herman from './cardPicture/herman.png';
+import calypso from './cardPicture/calypso.png';
+import helmut from './cardPicture/helmut.png';
+import ezmo from './cardPicture/ezmo.png';
+import bob from './cardPicture/bob.png';
+import flies from './cardPicture/flies.png';
+import sharkonaut from './cardPicture/sharkonaut.png';
+import nowifi from './cardPicture/nowifi.png';
+import diana from './cardPicture/diana.png';
+import brandon from './cardPicture/brandon.png';
+import creaturePool from './cardPicture/creature_pool.png';
+import nutzians from './cardPicture/nutzians.png';
+import alien from './cardPicture/alien.png';
+import brusselsSprout from './cardPicture/brussels_sprout.png';
+import yetirminator from './cardPicture/yetirminator.png';
+import propzilla from './cardPicture/propzilla.png';
+import cthulhu from './cardPicture/chtuhlu.png';
+import gordon from './cardPicture/gordon.png';
+import futura from './cardPicture/futura.png';
+import maga from './cardPicture/maga.png';
+import fawkes from './cardPicture/fawkes.png';
+import hyperion from './cardPicture/hyperion.png';
+import minus1 from './cardNumber/-1.png';
+import zero from './cardNumber/0.png';
+import one from './cardNumber/1.png';
+import two from './cardNumber/2.png';
+import three from './cardNumber/3.png';
+import four from './cardNumber/4.png';
+import five from './cardNumber/5.png';
+import six from './cardNumber/6.png';
+import seven from './cardNumber/7.png';
+import eight from './cardNumber/8.png';
+import nine from './cardNumber/9.png';
+import ten from './cardNumber/10.png';
+import eleven from './cardNumber/11.png';
+import twelve from './cardNumber/12.png';
+import thirteen from './cardNumber/13.png';
+import fourteen from './cardNumber/14.png';
+import fifthteen from './cardNumber/15.png';
+
+export const gameCards = {
+    '-1.0': {
+        score: -1,
+        picture: lola,
```

Comment:
du coup la propriété picture c'est du base64 ?

Reply:
non en fait create-react-app qui utilise webpack derrière détecte que le fichier importé est une image et l'ajoute au bundle et remplace l'import par le chemin vers l'image :
https://create-react-app.dev/docs/adding-images-fonts-and-files